code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -471,7 +471,7 @@
 	name = "Slime Potion 2"
 	id = "m_potion2"
 	required_container = /obj/item/slime_extract/lightpink
-	required_reagents = list(/datum/reagent/toxin/plasma = 1)
+	required_reagents = list(/datum/reagent/medicine/strange_reagent = 5)
```

Comment:
Strange reagent is a pretty big nerf, this stuff is very difficult to get.

Reply:
Correct